_rules/audio-or-video-avoids-automatically-playing-audio-x0paj4.md

```diff
@@ -0,0 +1,206 @@
+
+## Accessibility Support
+
+Some major browsers do not automatically play the 'video' unless the 'video' is muted. For these, the rule is never applicable. But, for UA that autoplay not muted video, the rule is still applicable.
+
+## Background
+
+The [instruments][instrument] used to pass this rule (if any), must meet all level A Success Criteria in order to fully satisfy [Success Criterion 1.4.2 Audio Control][sc142]. This means the [instrument][] to control the sound should be visible and accessible in order to be effective and usable by all kinds of users. If the [instrument][] is hidden to some users, it is possible to pass this rule but still not satisfy [Success Criterion 1.4.2 Audio Control][sc142]. These extra requirements are left out of this rule, and should be tested separately.
+
+### Bibliography
+
+- [Understanding Success Criterion 1.4.2: Audio Control](https://www.w3.org/WAI/WCAG21/Understanding/audio-control.html)
+- [Making Audio and Video Media Accessible](https://www.w3.org/WAI/media/av/)
+- [Failure of Success Criterion 1.4.2 for absence of a way to pause or stop an HTML5 media element that autoplays](https://www.w3.org/WAI/WCAG21/Techniques/failures/F93)
+- [G170: Providing a control near the beginning of the Web page that turns off sounds that play automatically](https://www.w3.org/WAI/WCAG21/Techniques/general/G170)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+This `audio` element has its default controls as an [instrument][] to pause, stop, and mute the audio.
+
+```html
+<audio src="/test-assets/moon-audio/moon-speech.mp3" autoplay controls></audio>
+```
+
+#### Passed Example 2
+
+This `video` element has its default controls as an [instrument][] to pause, stop mute the audio.
+
+```html
+<video autoplay controls>
+	<source src="/test-assets/rabbit-video/video.mp4" type="video/mp4" />
+	<source src="/test-assets/rabbit-video/video.webm" type="video/webm" />
+</video>
+```
+
+#### Passed Example 3
+
+This `video` element autoplays and has an [instrument][] to pause, stop, or mute the audio.
+
+```html
+<body>
+	<div id="video-container">
+		<!-- Video -->
+		<video id="video" autoplay>
+			<source src="/test-assets/rabbit-video/video.mp4" type="video/mp4" />
+			<source src="/test-assets/rabbit-video/video.webm" type="video/webm" />
+		</video>
+		<!-- Video Controls -->
+		<div id="video-controls">
+			<button type="button" id="play-pause" class="play">Play</button>
+			<button type="button" id="mute">Mute</button>
+		</div>
+	</div>
+	<script src="/test-assets/80f0bf/no-autoplay.js"></script>
+</body>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This `audio` element autoplays, lasts for more than 3 seconds, and does not have an [instrument][] to pause, stop, or mute the audio.
+
+```html
+<audio src="/test-assets/moon-audio/moon-speech.mp3" autoplay></audio>
+```
+
+#### Failed Example 2
```

Comment:
This example somewhat contradicts the idea that whether a failed example fails shouldn't depend on accessibility support. I'm a little worried this example could block implementations that run in Chromium. I know that could happen in axe Pro (although we have no tools testing this today). I'm pretty sure Trusted Tester would be impacted though, as it is tested in Edge.

So from that angle I'm not sure I would want this example in. On the other hand, not having a failed video example in it feels like a problem too. We could possibly wait with this rule until we have this "optional examples" idea worked out. A different option could be to modify the rule so that it isn't just about autocomplay, but more generally about media elements that start playing without user interaction within the first 3 second of the page loading? That feels like a reasonable improvement on this rule anyway, and we could then add videos that get turned on through a script during onload.